### ISSUE TITLE
sha1sum file is not gpg verified

### ISSUE THREAD

**Issue by modelprinting** [OutCast3k/coinbin#179]
I am downloading the coinbin.zip file and running on my local drive for security reasons. I notice that there is a "sha1sum" file with checksums for the contents of the zip file. Is there anyway this can be signed with a gpg key so i can make sure the checksum file hasn't been tampered with by a hacker? Is there any other way to gpg verify the zip file in some way?

**Comment by yottalogical** [comment]
You could always compare the sha1sum file to the one on GitHub to make sure it hasn't been tampered with.

**Comment by monperrus** [comment]
yes! it would be great that the [Github release page](https://github.com/outcast3k/coinbin/releases) also contains GPG signatures. thanks :)
